lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_version_helper.rb

```diff
@@ -1,18 +1,40 @@
 module Fastlane
     module Helpers
+      # A module containing helper methods to manipulate/extract/bump iOS version strings in xcconfig files
+      #
       module IosVersionHelper
+        # @!group The indices for various parts of the version parts array.
```

Comment:
This is a different heading than on Android – it kind of helps clarify things, though same note applies for your consideration.

Reply:
I actually ended up removing it, as (1) it doesn't add much value in the generated doc, and (2) it will likely disappear anyway once I get to https://github.com/wordpress-mobile/release-toolkit/issues/203